File: 2021/junior/python/j1.py
import sys

def j1():
    for line in sys.stdin:
        b = int(line.strip())
        print(5 * b - 400)
        diff = b - 100
        if diff == 0:
            print('0')
        elif diff > 0:
            print('-1')
        else:
            print('1')

if __name__ == '__main__':
    j1()

File: 2021/junior/cpp/j1.cpp
#include <iostream>

int main()
{
	int b{ };
	std::cin >> b;

    std::cout << (5 * b - 400) << '\n';

    int diff { b - 100 };
    if (diff == 0)
        std::cout << "0" << '\n';
    else if (diff > 0)
        std::cout << "-1" << '\n';
    else
        std::cout << "1" << '\n';
	return 0;
}